apps/ssr-tests-v9/src/utils/generateEntryPoints.test.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as tmp from 'tmp';
+
+import { getImportsFromIndexFile } from './getImportsFromIndexFile';
+import { generateEntryPoints } from './generateEntryPoints';
+
+jest.mock('./getImportsFromIndexFile');
```

Comment:
on the other hand, probably to better cover this functionality it would better approach if we do one of following:

1. mock the source of truth for those story paths

something like following:
![image](https://user-images.githubusercontent.com/1223799/172458326-6205db9f-f421-421e-9158-c45ce8a26790.png)


2. pass the stories glob to configuration object

![image](https://user-images.githubusercontent.com/1223799/172458852-e01a3a48-0e28-4437-a8dc-c7bffcf14626.png)

IMO this is better approach:
- more pure 
- no need mocking in the test
- would better test if glob works ( in current situation when I tested this solution they dont - I might be missing something )

WDYT ?

Reply:
Better to not have mocks 👍  I changed the idea a bit in ceba76f, WDYT?